docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -25,47 +25,50 @@ variables and shared common causes.
+- direction (ex: positive or negative effect of increased exposure), 
 - shape (ex: linear, U-shaped, etc.), 
 - magnitude (ex: large or small), 
 - and statisical certainty 
 
-of the association between the risk factor and the outcome. 
+of the effect the risk factor has on the outcome. 
 
 The risk effect may be measured in both absolute (ex: risk difference) or 
 relative terms (ex: relative risk). See the 
-:ref:`measures of excess risk page <excess_risk_measures>` for more 
+:ref:`measures of risk page <measures_of_risk>` for more 
 information on such measures.
 
-Risk effects are typically estimated in epidemiologic studies that assess
-how different levels of :ref:`risk exposures <risk_exposure>` affect some 
-outcome variable such as an incidence or mortality rate.
-There are several factors that may bias the quantification of a risk effect
-in epidemiologic studies (including influence by confounding), 
-and causal inference from such studies may be inhibited by such limitations.
-See the page on :ref:`causality <general_dags>` for more information on the topic.
+To implement a risk effect in simulation we also need to know how to
+*delete* that effect from baseline population-level rates of the outcome.
+This is done with a :ref:`measure of population impact <measures_of_impact>`.
 
 Risk effect in GBD
 ^^^^^^^^^^^^^^^^^^
 
-The measure of risk effect in GBD is usually reported in relative terms, namely
-relative risk. It describes the relative relationship between the risk of
-disease Y in the presence of agent X versus in absence of X. Mathematically,
-it's calculated by dividing the rate of some outcome (such as incidence or mortality
-rate) among the population exposed to a risk by the outcome rate in the unexposed population.
-For example, if there are A incident cases and B person-years in exposed group;
-C incident cases and D person-years in the unexposed group, then the relative risk
-(rate ratio) equals :math:`\frac{A/B}{C/D} = \frac{AD}{BC}`. The "unexposed" or "reference" group
-in the GBD-estimated relative risks is always the TMREL for that risk. For GBD 
-risk factors with continuous risk exposures, the GBD-estimated relative risks
-represent the relative risk associated with a defined unit increase in risk exposure
-above the TMREL. This unit increase may be more than a single unit change in risk
+GBD reports risk effects using relative risks (RRs) and population attributable fractions (PAFs).
+A relative risk describes the *multiplier* on the risk of the outcome
+caused by exposure.
+
+For GBD risk factors with categorical risk exposures,
+GBD reports an estimated relative risk for each level of risk exposure,
+relative to the TMREL for that risk.
+
+For most GBD risk factors with continuous risk exposures (see below for exceptions starting in GBD 2021),
+GBD reports an estimated relative risk *per unit increase* in risk exposure above the TMREL.
+This unit increase may be more than a single unit change in risk
```

Comment:
I know you didn't add this line, but I find it confusing. How can a unit be more than a unit? Do you know of any examples of what this means?